package.json

```diff
@@ -24,6 +24,7 @@
   ],
   "scripts": {
     "build": "npm run tsc.scripts && npm run tsc.prod && npm run rollup.prod.ci",
+    "build:esbuild": "git clean -fd && npm run clean && npm run build && node esbuild.js",
```

Comment:
haha good point - I'm good w/ `build.esbuild` although I must admit to an aesthetic preference for `:` - maybe because it's taller and looks more balanced? not important in any case 😆